Editor/CoreAPI/CoreApi.cs

```diff
@@ -36,6 +36,12 @@ public CoreApi()
         {
             _settingsStore = new SettingsStore(_fileWrapper, settingsFilePath: Paths.PluginSettingsFile);
             Bootstrapper.Initialize();
+        }        
```

Comment:
Remove? Now that we don't use this in the sample game

Reply:
Removing.